django/utils/crypto.py

```diff
@@ -17,7 +18,11 @@
 
 # Use the system PRNG if possible
 try:
-    random = random.SystemRandom()
+    if sys.version_info >= (3, 6):
+        import secrets
+        random = secrets.SystemRandom()
+    else:
+        random = random.SystemRandom()
```

Comment:
Could we try importing `secrets` with `from __future__ import absolute_imports` instead and fallback to `random` on `ImportError`?